packages/backend-defaults/src/entrypoints/urlReader/lib/AzureBlobStorageUrlReader.ts

```diff
@@ -44,7 +44,7 @@ export function parseUrl(url: string): { path: string; container: string } {
   const parsedUrl = new URL(url);
   const pathSegments = parsedUrl.pathname.split('/').filter(Boolean);
 
-  if (pathSegments.length < 2) {
+  if (pathSegments.length < 1) {
```

Comment:
Just to double-check 🙂, is the path optional?

Reply:
No it is not optional in `pathSegments` 0 index will be the container as per the azure blob storage url format and I had add check for the container being present in URL. Hence it is `<1`